packages/dashboard/src/components/SubwayNav/SubwayNav.styles.tsx

```diff
@@ -1,154 +1,94 @@
+import { DefaultFontStyles, keyframes } from 'office-ui-fabric-react';
+
 import { ISubwayNavStyles, ISubwayNavStyleProps } from './SubwayNav.types';
-import { DefaultPalette, DefaultFontStyles } from 'office-ui-fabric-react';
+import { ISubwayNavNodeProps, SubwayNavNodeState } from './SubwayNode.types';
+import { subwayColors } from './SubwayNode.styles';
+
+const itemSpacing = 36;
+const lineMargin = 4;
+const emOffset = 1.22;
 
-const regularFontSize = 14;

```

Comment:
Use FontSizes
https://fluentfabric.azurewebsites.net/#/typography

Reply:
I am, this was a deleted line. I'm using the DefaultFontStyles.

Reply:
Haha! Whoops!